projects/packages/videopress/src/client/block-editor/blocks/video/components/privacy-and-rating-panel/index.native.tsx

```diff
@@ -0,0 +1,170 @@
+/**
+ *External dependencies
+ */
+import { useNavigation } from '@react-navigation/native';
+import { PanelBody, SelectControl, ToggleControl, BottomSheet } from '@wordpress/components';
+import { useState, useCallback } from '@wordpress/element';
+import { __, _x } from '@wordpress/i18n';
+import { Icon, chevronRight } from '@wordpress/icons';
+/**
+ * Internal dependencies
+ */
+import {
+	VIDEO_PRIVACY_LEVELS,
+	VIDEO_PRIVACY_LEVEL_PRIVATE,
+	VIDEO_PRIVACY_LEVEL_PUBLIC,
+	VIDEO_PRIVACY_LEVEL_SITE_DEFAULT,
+	VIDEO_RATING_G,
+	VIDEO_RATING_PG_13,
+	VIDEO_RATING_R_17,
+} from '../../../../../state/constants';
+/**
+ * Types
+ */
+import type { VideoControlProps } from '../../types';
+import type React from 'react';
+
+/**
+ * Sidebar Control component.
+ *
+ * @param {VideoControlProps} props - Component props.
+ * @returns {React.ReactElement}    Component template
+ */
+export default function PrivacyAndRatingPanel( {
+	attributes,
+	setAttributes,
+	privateEnabledForSite,
```

Comment:
I noted this attribute is always `undefined`. But I understand that, as mentioned in the PR's description, this is due to the lack of metadata synchronization via the `useSyncMedia` hook. We'll need to test it once we implement that hook in the native version.